simulations/agents/uniswap_agent.py

```diff
@@ -183,7 +182,7 @@ def _quote_price(change_token_1):
                     maxiter=5,
                 )
                 change_token_1 = sol.root
-            except eth_abi.exceptions.ValueOutOfBounds:
+            except:  # noqa: E722
```

Comment:
Just so I know, how come this is reverted?

Reply:
One of the revert error was that the trade quoted was negative and the eth_abi could not encode the data. That was the eth_abi.exceptions.ValueOutOfBounds. 

Another revert error was from the `call()` and it was an empty error. I don't have full control over the iterations of the  `root_scalar` algorithm. In this case, when it crashes I just return an empty transaction